### PR TITLE
feat : added contributors and issue templates

### PR DESCRIPTION
**Pull request by Ayush7614**
This PR Contains

* Added Issue Templates
* Added Contributors in README section

In this project there is no issue templates was there so i adding all the issue templates that can be very helpfull for open any new issue and also for new comers

### CODE REVIEW

**Comment by Ayush7614** [comment]
cc : @t-fine

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      label: Expected behavior.
+      description: "Enter a clear and concise description of what you expected to happen"
+  - type: textarea
+    attributes:
+      label: Screenshots.
+      description: "If applicable, add screenshots to help explain your problem"
+  - type: input
+    attributes:
+      label: Operating environment
+      description: "What operating system and version are you using? If you are running it on Cloud/Cluster do mention about that."
```
Comment (clementkng) [inline .github/ISSUE_TEMPLATE/bug-report.yml:36]: Perhaps more concisely: "Are you running on a cloud/cluster?"

File: .github/ISSUE_TEMPLATE/Other.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+name: Other
+about: Something else
+
+---
+
+<!-- Bug reports and Feature requests must use other templates, or will be closed -->
+<!-- Please ask questions on StackOverflow or the webpack Gitter (https://gitter.im/webpack/webpack). -->
```
Comment (clementkng) [inline .github/ISSUE_TEMPLATE/Other.md:8]: We should remove this line since it isn't specific to Open Horizon.

**Review by clementkng — COMMENTED** [review]
Thanks for contributing!

File: .github/ISSUE_TEMPLATE/Other.md (hunk 2 above)
Comment (Ayush7614) [reply .github/ISSUE_TEMPLATE/Other.md]: Done @clementkng

File: .github/ISSUE_TEMPLATE/bug-report.yml (hunk 1 above)
Comment (Ayush7614) [reply .github/ISSUE_TEMPLATE/bug-report.yml]: I change whole template so new comers and everyone can understand very well

**Comment by Ayush7614** [comment]
> Thanks for contributing!

cc @clementkng

**Review by joewxboy — COMMENTED** [review]
Not sure that the check-broken-links.md file is needed.  It is there in support of the GHA that checks documentation for broken hyperlinks.  We can probably remove this file.

Also, to enable these custom issue templates, you may want to add a `config.yml` file in the repo root similar to this: https://github.com/open-horizon/open-horizon.github.io/blob/master/.github/config.yml. to enable a menu and activate the templates.

**Comment by Ayush7614** [comment]
cc @joewxboy

**Comment by Ayush7614** [comment]
Cc @joewxboy

**Comment by joewxboy** [comment]
@Ayush7614 Please link this PR to the related issue, and prefix the PR title with `Issue-##: `

**Comment by Ayush7614** [comment]
> @Ayush7614 Please link this PR to the related issue, and prefix the PR title with `Issue-##: `

@joewxboy Sorry I am not open any issue for this time for this PR but from next time i am always opening issue for the any changes and PR and please review and merged my code

**Review by t-fine — APPROVED** [review]

**Comment by Ayush7614** [comment]
cc @clementkng @joewxboy

**Comment by Ayush7614** [comment]
CC @joewxboy

**Review by clementkng — APPROVED** [review]

**Comment by t-fine** [comment]
With three approvals, I'm going to go ahead and merge this is.

**Pull request merged**